crates/bdk/src/wallet/mod.rs

```diff
@@ -95,6 +95,90 @@ pub struct Wallet<D = ()> {
     secp: SecpCtx,
 }
 
+/// A structure to update [`Wallet`].
```

Comment:
```suggestion
/// An update to [`Wallet`].
``` 

There is no need to refer to it as a structure. Anyone can see it is a `struct`.